Sources/Fisticuffs/Scheduler/RunLoop+Scheduler.swift

```diff
@@ -0,0 +1,38 @@
+//  The MIT License (MIT)
+//
+//  Copyright (c) 2021 theScore Inc.
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files (the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions:
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+
+import Foundation
+
+extension RunLoop: Scheduler {
+    public func schedule(_ action: @escaping () -> Void) {
+        if #available(iOS 10.0, *) {
```

Comment:
Personally I don't see a reason to keep supporting iOS 10.0 but at the same time this is a small maintenance cost so I didn't update the minimum version

Reply:
Personally I'd lean towards upping it as opposed to adding code that will likely never be useful for anyone but I'm not sure if there's other overhead involved with that so I'm fine to leave it like this if you don't feel it's worthwhile to pursue that.

Reply:
Yea I'd rather just leave the version for now especially since it's out of the scope of this PR. We can revisit this when performing maintenance in this repo